### PR TITLE
Fix race conditions pt1

### PR DESCRIPTION
**Pull request by benma**
This fixes some potential race conditions. More to come.

### CODE REVIEW

File: backend/coins/btc/account.go [hunk 1]
```diff
@@ -99,13 +99,15 @@ type Account struct {
...
+	feeTargets   []*FeeTarget
+	feeTargetsMu locker.Locker
 	// Access this only via getMinRelayFeeRate(). sat/kB.
 	minRelayFeeRate   *btcutil.Amount
 	minRelayFeeRateMu sync.Mutex
 
 	// true when initialized (Initialize() was called).
-	initialized bool
+	initialized   bool
+	initializedMu locker.Locker
```
Comment (Beerosagos) [inline backend/coins/btc/account.go:110]: could this be an `atomic.bool` instead?

File: backend/coins/btc/account.go [hunk 2]
```diff
@@ -99,13 +99,15 @@ type Account struct {
 	activeTxProposal     *maketx.TxProposal
 	activeTxProposalLock locker.Locker
 
-	feeTargets []*FeeTarget
+	feeTargets   []*FeeTarget
+	feeTargetsMu locker.Locker
```
Comment (Beerosagos) [inline backend/coins/btc/account.go:103]: I see that in this structure we use `Locker` or `Mutex` in different places. Would be possible to uniform it and use Lockers everywhere? Apart from this: `feeTargetsMu` suggests that it is a Mutex. Maybe change to `feeTargetsLock` instead? Same for `initializedMu`

**Review by Beerosagos — APPROVED** [review]
tACK with small nits

File: backend/coins/btc/account.go (hunk 1 above)
Comment (benma) [reply backend/coins/btc/account.go:110]: No as the whole Initialize function needs to be serialized, not just writing to and reading from this variable.

File: frontends/web/src/api/account.ts [hunk 3]
```diff
@@ -99,7 +99,7 @@ export type TSigningConfiguration = {
     ethereumSimple: TEthereumSimple;
 }
 
-export interface ISigningConfigurationList {
+export type ISigningConfigurationList = null | {
```
[the remark this answers is not in the document]
Comment (benma) [reply frontends/web/src/api/account.ts]: Done

File: backend/coins/btc/account.go (hunk 2 above)
Comment (benma) [reply backend/coins/btc/account.go]: Done

**Pull request merged**